homeassistant/components/plugwise/select.py

```diff
@@ -54,6 +54,15 @@ class PlugwiseSelectEntityDescription(
         options_key="regulation_modes",
         device_class="plugwise__regulation_mode",
     ),
+    PlugwiseSelectEntityDescription(
+        key="select_dhw_mode",
+        name="DHW mode",
+        icon="mdi:shower",
+        entity_category=EntityCategory.CONFIG,
+        command=lambda api, loc, opt: api.set_dhw_mode(opt),
+        current_option_key="dhw_mode",
+        options_key="dhw_modes",
```

Comment:
Should this have a custom device class and translations for its options?

Reply:
These are the provided modes:
![image](https://user-images.githubusercontent.com/11290930/196608952-26110924-5bce-4963-9e6f-3393fb2b2a42.png)

So, yes. might be a good idea.

Reply:
Added.